src/components/routingActions.js

```diff
@@ -20,7 +20,10 @@ export const getRedirectParams = (action, actionParams) => {
     case 'afspraak-maken':
       return {path: 'afspraak-maken'};
     case 'resume':
-      return {path: `stap/${actionParams.next_step}`};
+      return {
+        path: `stap/${actionParams.step_slug}`,
```

Comment:
A different parameter name was being used in the backend: https://github.com/open-formulieren/open-forms/blob/master/src/openforms/submissions/views.py#L187